relation/core.md

```diff
@@ -23,14 +23,28 @@ The standard error is 3 standard deviation of the mean of the sample.
  
  Dispersion of the values of a random variable around its expected value.
 
-### relation:MarginOfError {#MarginOfError}
+### relation:Confidence {#MarginOfError}
  
- The margin of error is the range of values below and above the sample statistic in a confidence interval.
+Use `Confidence` to specify the level of uncertainty in the estimate. It can be used multiple times,
+in which case add `dcterms:type` to each dimension relation to specify the type of confidence 
+(Margin of Error, Confidence Interval, etc.).
  
-#### relation:MarginOfErrorUpperBound {#MarginOfErrorUppoerBound}
-
-  In case of asymmetric confidence intervals, the upper bound of the margin of error.
-  
-#### relation:MarginOfErrorLowerBound {#MarginOfErrorLowerBound}
-
-  In case of asymmetric confidence intervals, the lower bound of the margin of error.
+#### relation:ConfidenceUpperBound and relation:ConfidenceUpperBound {#ConfidenceBounds}
```

Comment:
Shouldn't this be `relation:ConfidenceUpperBound` and `relation:ConfidenceLowerBound`?

Reply:
Of course!